cmd/traffic_cache_tool/CacheTool.cc

```diff
@@ -1734,55 +1728,6 @@ Cache::~Cache()
   for (auto *span : _spans)
     delete span;
 }
-/* --------------------------------------------------------------------------------------- */
-#if 0
-Errata
-Span::Initialize()
-{
-  Errata zret;
-
-  for (auto *strp : _stripes) {
-    std::cout << "strpe start=============" << strp->_start << std::endl;
-  }
-  auto skip = CacheStoreBlocks::SCALE;
-  // successive approximation to calculate start
-  auto start = skip;
-  uint64_t l;
-  for (int i = 0; i < 3; i++) {
-    l = _len - (start - skip);
-    if (l >= MIN_VOL_SIZE) {
-      header_len = sizeof(ts::SpanHeader) + (l / MIN_VOL_SIZE - 1) * sizeof(CacheStripeDescriptor);
-    } else {
-      header_len = sizeof(ts::SpanHeader);
-    }
-    start = skip + header_len;
-  }
-  // num_usable_blocks = ((_len * STORE_BLOCK_SIZE) - (start - skip)) >> STORE_BLOCK_SHIFT;
-  header_len       = ROUND_TO_STORE_BLOCK(header_len);
-  std::cout << "start " << start << "len " << _len << " header length " << header_len << std::endl;
-  num_usable_blocks = (_len - (start - skip)) >> STORE_BLOCK_SHIFT;
-  return zret;
-}
-#endif
-
-#if 0
-Errata
-Span::create_stripe(int number, off_t size_in_blocks, int scheme)
```

Comment:
Where did this functionality go?

Reply:
we don't need this anymore. I reused the allocation logic you had for the stripe creation and then have a separate method for initializing the meta